### PR TITLE
Add Timetable Items

### LINKED ISSUE #627: Timetable Items
## Kind (Required)

- Improvement

 ## Overview (Required)

- Use LazyColums to display items in the timetable.
image

 ## Links	

-

### PR DESCRIPTION
**Pull request by r4wxii**
## Issue
- #627 

## Overview (Required)
- Use LazyColums to display items in the timetable.

## Links
-

### CODE REVIEW

**Comment by takahirom** [comment]
👀

**Review by takahirom — COMMENTED** [review]
I'm a little concerned about the location of the TimetableListItemState definition and the name of the Composable function, but everything else looks good, so let me know when this PR is ready!

**Comment by r4wxii** [comment]
I implemented Timetable Items and fixed it that was commented 🙋‍♂️

**Comment by r4wxii** [comment]
When #679 was merged, I would like to additional implementations including the changes.

File: model/src/commonMain/kotlin/io/github/droidkaigi/feeder/TimetableItem.kt [hunk 1]
```diff
@@ -62,4 +64,9 @@ sealed class TimetableItem(
         asset = asset,
         levels = levels,
     )
+
+    val startsTimeString: String by lazy {
+        val localDate = startsAt.toLocalDateTime(TimeZone.currentSystemDefault())
+        "${localDate.hour}".padStart(2,'0') +  ":" + "${localDate.minute}".padStart(2,'0')
```
Comment (takahirom) [inline model/src/commonMain/kotlin/io/github/droidkaigi/feeder/TimetableItem.kt:70]: Can you apply format? 🙏 

```suggestion
        "${localDate.hour}".padStart(2, '0') + ":" + "${localDate.minute}".padStart(2, '0')
```

File: uicomponent-compose/timetable2021/build.gradle [hunk 2]
```diff
@@ -11,16 +11,25 @@ apply from: rootProject.file("gradle/android.gradle")
...
+        // https://github.com/Kotlin/kotlinx.coroutines/issues/2023
+        exclude 'META-INF/AL2.0'
+        exclude 'META-INF/LGPL2.1'
+    }
 }
 
 dependencies {
     api projects.uicomponentCompose.core
     api projects.model
+    implementation 'androidx.test.ext:junit-ktx:1.1.3'
```
Comment (takahirom) [inline uicomponent-compose/timetable2021/build.gradle:32]: Can you add Dep dependency? 🙏

File: uicomponent-compose/timetable2021/src/androidTest/java/io/github/droidkaigi/feeder/timtable2021/TimetableItemTest.kt [hunk 3]
```diff
@@ -0,0 +1,79 @@
+package io.github.droidkaigi.feeder.timtable2021
+
+import androidx.activity.ComponentActivity
+import androidx.compose.ui.test.assertHasClickAction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.github.droidkaigi.feeder.core.theme.ConferenceAppFeederTheme
```
Comment (takahirom) [inline uicomponent-compose/timetable2021/src/androidTest/java/io/github/droidkaigi/feeder/timtable2021/TimetableItemTest.kt:9]:
```suggestion
```

File: uicomponent-compose/timetable2021/src/androidTest/java/io/github/droidkaigi/feeder/timtable2021/TimetableItemTest.kt [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import io.github.droidkaigi.feeder.fakeTimetableContents
+import io.github.droidkaigi.feeder.timetable2021.TimetableItem
+import io.github.droidkaigi.feeder.timetable2021.TimetableItemState
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class TimetableItemTest {
```
Comment (takahirom) [inline uicomponent-compose/timetable2021/src/androidTest/java/io/github/droidkaigi/feeder/timtable2021/TimetableItemTest.kt:18]: 👍

File: uicomponent-compose/timetable2021/src/main/java/io/github/droidkaigi/feeder/timetable2021/TimetableScreen.kt [hunk 5]
```diff
@@ -114,9 +125,14 @@ private fun TimetableScreen(
                 ) { page ->
                     val selectedTab = TimetableTab.values()[page]
                     TimetableList(
-                        TimetableListState(
-                            state.timeTableContents.getDayTimetableItems(selectedTab.day)
-                        )
+                        state = TimetableListState(
+                            state.timeTableContents
```
Comment (takahirom) [inline uicomponent-compose/timetable2021/src/main/java/io/github/droidkaigi/feeder/timetable2021/TimetableScreen.kt:129]: How about using named argment? 🙏 
`timetableItems = state.timeTableContents`

**Review by takahirom — APPROVED** [review]
LGTM!
Please check the small format issues 🙏

**Comment by r4wxii** [comment]
I fixed it 🙇‍♂️

**Review by takahirom — APPROVED** [review]
Thank you for addressing my comments. 💯 
LGTM!

**Pull request merged**